src/git.rs

```diff
@@ -67,8 +65,19 @@ fn path_command(path: impl AsRef<Path>, subcommand: &[&str]) -> Option<PathBuf>
             .and_then(std::char::from_u32),
         Some('\n')
     );
-    let toplevel = OsString::from_vec(output.stdout);
-    Some(toplevel.into())
+    Some(path_from_output(output.stdout))
+}
+
+#[cfg(unix)]
+fn path_from_output(output: Vec<u8>) -> PathBuf {
+    use std::ffi::OsString;
+    use std::os::unix::prelude::OsStringExt;
+    OsString::from_vec(output).into()
+}
+
+#[cfg(all(not(unix)))]
+fn path_from_output(output: Vec<u8>) -> PathBuf {
+    String::from_utf8(output).ok().unwrap().into()
```

Comment:
Is the ok really useful if you are unwrapping anyway?

Reply:
You won't see the error context this way, so I'd rather see the `ok()` dropped as well :)